test/test_config.rb

```diff
@@ -473,6 +473,10 @@ def test_run_hooks_on_restart_hook
     assert_run_hooks :on_restart
   end
 
+  def test_run_hooks_on_stopped_hook
+    assert_run_hooks :on_stopped
+  end
```

Comment:
Given the change above, the test should probably mirror [`test_on_booted`](https://github.com/puma/puma/blob/f027b319bcc72db27841d389d1efc6fc7d921634/test/test_integration_single.rb#L52-L59) instead.